template/.devcontainer/manual-setup-deps.py

```
@@ -0,0 +1,120 @@
+                sync_command,
+                check=True,
+                env=uv_env,
+            )
+        elif env.package_manager == PackageManager.PNPM:
+            pnpm_command = ["pnpm", "install", "--dir", str(env.path)]
+            if not env_skip_check_lock:
+                pnpm_command.append("--frozen-lockfile")
+            if is_windows:
+                pwsh = shutil.which("pwsh") or shutil.which("powershell")
+                if not pwsh:
+                    raise FileNotFoundError("Neither 'pwsh' nor 'powershell' found on PATH")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)